community/gsoc_responsibilities.md

```diff
@@ -2,7 +2,9 @@ Responsibilities of coala’s GSoC Participants
 =============================================
 
 This Document hold the responsibilities we expect coala’s GSoC participants to
-fulfil.
+fulfil.\
```

Comment:
please remove the trailing `\` here.
Then it is ready to be merged.
Thanks

Reply:
Add a blank line.

Reply:
@jayvdb Done changes. Up for review. Thanks!